ui/lib/kv/addon/components/kv-patch-editor.hbs

```diff
@@ -0,0 +1,165 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<form {{on "submit" this.submit}} ...attributes>
+  <div class="flex column-gap-16 has-top-padding-s">
+    <Hds::Form::Label @controlId="newKey" class="one-fourth-width">
+      Key
+    </Hds::Form::Label>
+    <Hds::Form::Label @controlId="newValue" class="three-fourths-width">
+      Value
+    </Hds::Form::Label>
+  </div>
+
+  {{! Rows for existing keys (includes new rows after user clicks "Add") }}
+  {{#each this.patchData as |KV idx|}}
```

Comment:
anywhere you see `KV` that refers to the `class Kv` here 

https://github.com/hashicorp/vault/blob/50b409111444e01bf2729937e5a5e67521b49f8f/ui/lib/kv/addon/components/kv-patch-editor.js#L18

I hoped that capitalizing the variables related to the class made it easier to trace where they came from, but let me know if that's unhelpful!